src/docs/linea.mdx

```diff
@@ -0,0 +1,192 @@
+                    tooltip: 'How the smart contract on the target chain is called. In both automatic and manual claiming we are waiting for a central unit to add the message hashes on the target chain (~17 minutes)',
+                    value: 'Manual'
+                }
+            ]
+        },
+        {
+            title: 'L2 → L1',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: "The time it takes for a message to be made available on Ethereum after being included in an Linea block and sequenced to L1. Messages can only be claimed after ZK proof has been submitted on L1 that verifies it's been submitted on L2.",
+                    value: '1 to 32 hours'
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of executing a message on Ethereum after being sent from Linea. The fee for the prover is paid by the sender on L2.',
+                    value: '73 000 L2 gas'
+                },
+                {
+                    label: 'Delivery method',
+                    tooltip: 'How the smart contract on the target chain is called.',
+                    value: 'Manual'
+                },
+            ],
+        },
+    ]} />
+
+</Section>
+
+<Section title="OPCODEs">
+
+    <Legend />
+
+    Linea uses the London version of the EVM.
+
+    The following table encapsulates EVM OPCODE differences of the Rollup compared to the canonical Ethereum L1 implementation.
+
+    | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :--- | :-------------------| :--------------- | :-------------------- |
+    | 44 | PREVRANDAO (legacy DIFFICULTY) | `block.difficulty` | Returns a fixed number "2" | Get the output of the randomness beacon provided by the beacon chain <Modified /> |
+    | 5F | PUSH0 |  | Not supported yet! <br /><br /> Solidity version `0.8.20` or higher can only be used with an evm version lower than the default `shanghai`. <br /><br /> Versions up to `0.8.19` (included) are fully compatible | Places value 0 on the stack <Unsupported /> |
+
+</Section>
+
+<Section title="Precompiled Contracts">
+
+    > ⚠️ All Precompiled Contacts defined in the canonical Ethereum L1 implementation (until London hard fork) have the same behaviour on the Rollup, however, the execution of those Precompiles is not covered by the circuits and therefore not present in the Validity proof
+
+</Section>
+
+<Section title="System Contracts">
+
+    The Rollup has introduced the following system contracts.
+
+    | Address | Name | Description |
+    | :--- | :--- | :--- |
+    | <Copy value="0xC84832f69bFFbC1A94E44a157A342766E959ED27" label="0xC84...27" /> | <Reference label="PlonkVerifierFull" url="https://github.com/Consensys/linea-contracts/blob/main/contracts/verifiers/PlonkVerifierFull.sol" /> | Plonk verifier contract used by the Linea zkEVM rollup. |
```

Comment:
When we refer to "system contracts" in other rollups we are listing only the contracts that end users/developers can utilise. Think of it as "additional features" that are provided by the rollup except for the precompile and opcode functionality. 

Having that in mind, I do not think that the Plonk Verifier should be here.